pkg/util/apikey/balancer.go

```diff
@@ -0,0 +1,195 @@
+package apikey
+
+import (
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+	"sync"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// {PREFIX}_{FIELD}_{INDEX}(_{POSTFIX})?
+var DefaultPattern = regexp.MustCompile(
```

Comment:
The `DefaultPattern` is not used anywhere?
Might be better if we remove it.

Reply:
removed unused vars